data.sql

```diff
@@ -11,3 +11,24 @@ INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
 
 INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
     VALUES ('Devimon', '2017-05-12', 5, true, 11);
+
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
+    VALUES ('Charmander', '2020-02-08', 0, false, 11);
```

Comment:
- [ ] From the data provided in the project requirement document, the weight of `Charmander` is a negative decimal `-11.0` value. Kindly update your code.